### ISSUE TITLE
How and where to access html body in gatsby to avoid white space?

### ISSUE THREAD

**Issue by santhoshnumberone** [gatsbyjs/gatsby#28821]
I am having issue of large white spaces appearing on the right and the bottom of the web page. When testing for responsiveness of the web page.

I found a similar issues on stackoverflow

1) [White space showing up on right side of page when background image should extend full length of page [closed]](https://stackoverflow.com/questions/4617872/white-space-showing-up-on-right-side-of-page-when-background-image-should-extend)

2) [Website has strange whitespace on right side of the page when the browser is resized to a smaller window](https://stackoverflow.com/questions/4612307/website-has-strange-whitespace-on-right-side-of-the-page-when-the-browser-is-res?noredirect=1&lq=1)

Solution in both the posts are same

```
html, 
body {
    width: 100%;
    height: 100%;
    margin: 0;
    padding: 0;
    overflow-x: hidden; 
}
```
I don't know where exactly to add this in the `gatsby`, I found a post with a similar issue to mine regarding `gatsby` [How do I style the body/main background?](https://spectrum.chat/gatsby-js/general/how-do-i-style-the-body-main-background~c410949e-7bbc-46d5-8701-785668308341). Don't seem to understand what is the solution for this!

Found this post [How to add a dynamic class to body tag in Gatsby.js?](https://stackoverflow.com/questions/40995948/how-to-add-a-dynamic-class-to-body-tag-in-gatsby-js) got some idea about using `react-helmet`, how exactly I can use it?

Could anyone explain how I can set the `html body` in gatsby, to avoid this large white space?

**Comment by LekoArts** [comment]
Thank you for opening this!

You can create a `styles.css` file at the root of your project, then create a `gatsby-browser.js` at the root of your project and import the file:

```js
// in your gatsby-browser.js

import "./styles.css"
```

We're marking this issue as answered and closing it for now but please feel free to comment here if you would like to continue this discussion. We also recommend heading over to our [communities](https://www.gatsbyjs.org/contributing/community/#where-to-get-support) if you have questions that are not bug reports or feature requests. We hope we managed to help and thank you for using Gatsby!

**Comment by GammaWatt** [comment]
I came here looking for an answer for this exact issue, and you closed the question without any feedback as to whether or not the problem was actually solved. Very nice.

And yes, I have tried the proposed solution. I have modified everything, even directly from within the browser, and it is as if the blank bar is hardcoded into the site.
I have no issue when coding the html by hand, but when I render from gatsby, I cannot remove the empty space.
